content/articles/ordering-a-wildcard-certificate.markdown

```diff
@@ -0,0 +1,47 @@
+---
+title: Ordering a Wildcard SSL Certificate
+excerpt: How to order a wildcard SSL certificate with DNSimple.
+categories:
+- SSL Certificates
+---
+
+# Ordering a Standard SSL Certificate
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+Wildcard certificates allow you to secure all subdomains of a domain with a single certificate. Wildcard certificates can make certificate management easier in some cases. We still recommend non-wildcard certificates for most use cases.
```

Comment:
> Wildcard certificates allow you to secure all subdomains of a domain with a single certificate.

This is not technically true. It allows to secure an unlimited number of names that belong to a single subdomain level.

It's picky, but it's a key difference. We should also update the wording on https://support.dnsimple.com/articles/ssl-certificates-types/#ssl-certificates-by-secured-domains

Reply:
> We still recommend non-wildcard certificates for most use cases.

Reason?

Reply:
This initial part was lend from https://community.letsencrypt.org/t/acme-v2-and-wildcard-certificate-support-is-live/55579 😬 

>  It allows to secure an unlimited number of names that belong to a single subdomain level.

Will update that.

>  > We still recommend non-wildcard certificates for most use cases.
> 
> Reason?

When a wildcard certificate gets compromised an attacker can use the certificate to secure a possibly compromised domain. I'm happy to remove that.

Reply:
That makes sense. However, I think in the context of this support page it may be more confusing than helpful (especially when left broadly open as it is now).

I'd vote for remove it. Or you can change it into:

> Wildcard certificates can make certificate management easier in some cases. However, you should avoid them unless you have a specific need such as multi-tenant applications.

Reply:
✅